packages/netlify-cms-proxy-server/src/middlewares/joi/index.ts

```diff
@@ -136,6 +140,7 @@ export const defaultSchema = ({ path = requiredString } = {}) => {
               status: requiredString,
             }).required(),
           })
+          .xor('entry', 'dataFiles')
```

Comment:
Either `entry` (for backwards compatibility) or `dataFiles` must be included in the request, but not both